src/android/ApptentiveBridge.java

```diff
@@ -58,13 +59,24 @@
 	public ApptentiveBridge() {
 	}
 
-	   UnreadMessagesListener listener = null;
+    UnreadMessagesListener listener = null;
     
 	public boolean execute(final String action, JSONArray args, final CallbackContext callbackContext) throws JSONException {
 		ApptentiveLog.v("Executing action: %s", action);
 
-
 		if (action.equals(ACTION_DEVICE_READY)) {
+            final Activity currentActivity = cordova.getActivity();
+            if (currentActivity != null) {
+                currentActivity.runOnUiThread(new Runnable() {
+                    @Override
+                    public void run() {
+                        Apptentive.register(currentActivity.getApplication());
+                        ApptentiveInternal.getInstance().getRegisteredLifecycleCallbacks().onActivityCreated(currentActivity, null);
+                        ApptentiveInternal.getInstance().getRegisteredLifecycleCallbacks().onActivityStarted(currentActivity);
+                        ApptentiveInternal.getInstance().getRegisteredLifecycleCallbacks().onActivityResumed(currentActivity);
+                    }
+                });
+            }
```

Comment:
What if a customer already extended the Application object? We should handle that case right? Could it be as simple as checking to see if registration has already occurred?